packages/front-end/pages/settings/index.tsx

```diff
@@ -1576,6 +1577,20 @@ const GeneralSettingsPage = (): React.ReactElement => {
                     }}
                   />
                 </div>
+                <div>
+                  <label className="mr-1" htmlFor="toggle-require-reviews">
+                    Require approval to publish changes:
+                  </label>
+                </div>
+                <div>
+                  <Toggle
+                    id={"toggle-require-reviews"}
+                    value={!!form.watch("requireReviews")}
+                    setValue={(value) => {
+                      form.setValue("requireReviews", value);
+                    }}
+                  />
+                </div>
```

Comment:
This setting should be enterprise-only.  Need to define a new commercial feature for "require-approvals" in `license.ts` and assign it to the enterprise plan.